### ISSUE TITLE
Pickit Help

### ISSUE THREAD

**Issue by Greghmm** [kolton/d2bot-with-kolbot#1319]
OK, first let me start with this. I have done a ton of research on the order of things when it comes to writing pickit lines for the bot.  I understand what the different symbols and all mean.  My issue is that I have created a pickit line and it does not throw any errors in the game but in the D2Bot window it throws an error.  See below for both  pickit line and error.

[Type] == ring && [Quality] == rare # [FCR] >= 10 && ([FireResist] >= 10 || [LightResist] >= 10 || [ColdResist] >= 10 || [PoisonResist] >= 10) && ([MaxMana] >= 30 || [MaxHP] >= 30) && [Strength] >= 10 || [Dexterity] >= 10)

Pickit error! Line # 439 Entry: [Type] == ring && [Quality] == rare # [FCR] >= 10 && ([FireResist] >= 10 || [LightResist] >= 10 || [ColdResist] >= 10 || [PoisonResist] >= 10) && ([MaxMana] >= 30 || [MaxHP] >= 30) && [Strength] >= 10 || [Dexterity] >= 10) (greg.nip) Error message: missing ; before statement Trigger item: Ring

Could someone please help me understand why I am getting error.  I have looked at it 100 times and can't seem to see the issue.  Thank you!

**Comment by chanceplay** [comment]
Need opening "(" before "[Strength]"

**Comment by Greghmm** [comment]
SoB… Thank you! That's why I needed a fresh set of eyes.

**Comment by oband2** [comment]
@Greghmm 
Do u have a link for your research? would like to know more about pickit lines for the bot, thanks!

**Comment by Greghmm** [comment]
https://github.com/blizzhackers/pickits/blob/master/NipGuide.md/#nip-guide

**Comment by 74Thirsty** [comment]
Okay theres a few problems that i am having with pickit. The most important to resolve is that its not picking up white or socketed items. For example... I want a base for white with like +3 spirit +3 ce +3 revive and i dont care what kind of wand so i wrote it like this:
([name] == bonewand || [name] == grimwand || [name] == bonewand || [name] == petrifiedwand || [name] == ghostwand || [name] == lichwand || [name] == gravewand || [name] == polishedwand || [name] == yewwand || [name] == wand) # [skillbonespirit] == 3 && ([skillrevive]+[skillconfuse]+[skillamplifydamage]+[skilldecrepify]+[skillcorpseexplosion]+[skilllowerresist] >= 3) 


I should be able to instead use this right:
[type] == wand && [flag] != ethereal # [skillbonespirit] == 3 && ([skillrevive]+[skillconfuse]+[skillamplifydamage]+[skilldecrepify]+[skillcorpseexplosion]+[skilllowerresist] >= 3) 

 basically i just want +3 spirit and a curse or some ce on it too
                                

but it wont grab none of the white ones or socketed ones. 

It does however grab blus ones all day which are the one kind i am not interseted in those. I want to check all rares f0r  2\3 fcr etc  an plaing ones for 0 or 2 sockets with gg skills. 

And it isnt just this line thats messed up. It dont like to grab any white or grey items
